### PR TITLE
Remove user mark guard.

### PR DESCRIPTION
**Pull request by darina**

### CODE REVIEW

File: android/jni/com/mapswithme/maps/Framework.cpp [hunk 1]
```diff
@@ -643,7 +643,7 @@ Java_com_mapswithme_maps_Framework_nativeGetNameAndAddress(JNIEnv * env, jclass
 JNIEXPORT void JNICALL
 Java_com_mapswithme_maps_Framework_nativeClearApiPoints(JNIEnv * env, jclass clazz)
 {
-  UserMarkControllerGuard guard(frm()->GetBookmarkManager(), UserMarkType::API_MARK);
+  UserMarkNotifyGuard guard(frm()->GetBookmarkManager(), UserMarkType::API_MARK);
```
Comment (rokuz) [inline android/jni/com/mapswithme/maps/Framework.cpp]: UserMarkNotificationGuard ?

**Comment by rokuz** [comment]
DUDE JTTP

**Comment by darina** [comment]
DUDE JTTP

File: android/jni/com/mapswithme/maps/Framework.cpp (hunk 1 above)
Comment (darina) [inline android/jni/com/mapswithme/maps/Framework.cpp]: готово

**Pull request merged**